preload/supported-as-values.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<title>Test the supported value for &lt;link rel=preload as="..."&gt;</title>
+<meta name="timeout" content="long">
+<meta name="variant" content="?as=image&expected=1">
```

Comment:
Neat way to have multiple test cases in a single HTML!!

Reply:
Yea, found out about it recently, much more readable than having for-loops that create tests